### PR TITLE
Fix error screen and update README

### PR DESCRIPTION
**Pull request by cookta2012**
Fix bootstrap.lua error screen.
Update README.md instructions and progress

### CODE REVIEW

**Comment by cookta2012** [comment]
This is good for Merge

**Review by gucio321 — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -14,22 +14,23 @@ Current Features:
...
 * Gameplay (when the engine supports it)
 
 # Installation
 
 1. Download OpenDiablo2 to a folder (or clone the repo)
 1. Copy the data files over from the copy of Diablo II you have. You need to do at least one of the following:
-   1. If you have good old Diablo II (latest LOD patch required), copy the MPQs from the game's installation folder into the MPQs folder of this project (there should be 11 MPQs).
+   1. If you have good old Diablo II (latest LOD patch required), copy the MPQs from the game's installation folder into the DATA\MPQ-D2 folder of this project (there should be 11 MPQs).
+      * DO NOT DELETE THE `.loadorder` file from this folder, it should be a hidden file on most systems. 
    1. If you have Resurrected, copy Data/ and .build.info files from it into D2RData folder of this project. Speficially, the file structure should be like this:
```
Comment (DarthGandalf) [inline README.md:29]: update folder name

File: README.md [hunk 2]
```diff
@@ -14,22 +14,23 @@ Current Features:
...
-* Credits Screen (Can be done now)
-* New Hero Screen (Can be done now except for text input)
 * Gameplay (when the engine supports it)
 
 # Installation
 
 1. Download OpenDiablo2 to a folder (or clone the repo)
 1. Copy the data files over from the copy of Diablo II you have. You need to do at least one of the following:
-   1. If you have good old Diablo II (latest LOD patch required), copy the MPQs from the game's installation folder into the MPQs folder of this project (there should be 11 MPQs).
+   1. If you have good old Diablo II (latest LOD patch required), copy the MPQs from the game's installation folder into the DATA\MPQ-D2 folder of this project (there should be 11 MPQs).
```
Comment (DarthGandalf) [inline README.md:27]: can you use / ?
\ implies windows and works only on windows, but / works everywhere, including windows

**Pull request merged**